docs/src/api/class-elementhandle.md

```diff
@@ -743,6 +743,29 @@ content.
 ### option: ElementHandle.selectText.force = %%-input-force-%%
 ### option: ElementHandle.selectText.timeout = %%-input-timeout-%%
 
+## async method: ElementHandle.setChecked
+
+This method checks or unchecks an element by performing the following steps:
+1. Ensure that element is a checkbox or a radio input. If not, this method throws. If the element is already
```

Comment:
"If the element is already checked" -> "has correct checked state" or something?